eu/ja-jp/content/topics/end-user/ov-device-health-trbl.htm

```diff
@@ -420,12 +420,12 @@ <h3><a name="Android"></a>Android </h3>
                                                                     <ol>
                                                                         <li value="1">
                                                                             <p>
-ご利用のデバイスで<span class="uicontrol">［Settings（設定）］</span>に移動します。
+ご利用のデバイスで<span class="uicontrol">［設定］</span>に移動します。
```

Comment:
@pengyao-okta One more monolingual label; the previous 37 files are clean, though, so the trend is still good.